### DISCUSSION TITLE
Add DALLE-2 support

### DISCUSSION THREAD

**Discussion by ghost** [ThioJoe/Full-Stack-AI-Meme-Generator#3] [General]
Adding DALLE-2 support to the bot would be easy, considering an openai API key is already specified, and it would also mean that you would only need a OpenAI key to use the program. I will implement support for this feature and many others in a seperate fork.

**Comment by ThioJoe** [comment]
It already has DALLE-2 support, that's what the "openai" option is for image_platform 👍. I'll have to make that more clear.
